### PR TITLE
Remove background grey in header

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Check that the canvas now covers the entire viewport
- Check that service blocks now go under the header and all the way to the footer
- Check there is nothing broken in the search panel

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3732/
Test PASSed.

**Comment by hatched** [comment]
QA Issues:

- There is now a line between the search input and the midpoint:
screenshot from 2015-11-30 11 27 52

- Even though the content is visible through the header you can't actually click there to move anything.

I think that this branch is a little more complex than a few styling changes. Each component I believe also needs to be absolutely placed instead of there being a 'header' element that spreads across the whole top of the page.

**Comment by anthonydillon** [comment]
I agree, will pick this up on weds and see if I can get all the elements in a row. Thanks for the QA @hatched

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3757/
Test PASSed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3759/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3760/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3764/
Test PASSed.

**Comment by anthonydillon** [comment]
@hatched ready for another review. 

## Extra QA parts
- Moved header from the viewport to the top of the dom
- Moved one or two elements to top level nodes instead of being children
- Updated the key shortcuts panel with the new design [1]
- All z-index's now pull there index location from a z-index-list

**Comment by hatched** [comment]
Looks good, but I still can't click a service if it's under the header which is a little odd because I can see it. What if you positioned the elements outside of any header element so that the middle area was clear and click-through-able?

**Comment by anthonydillon** [comment]
Everything was clockmaker for me. Are you on Chrome? The midpoint element
isn't in the header. Do you mean the services when you click on search?

**Comment by hatched** [comment]
Yep in chrome.
- Add a service to the canvas.
- Drag that service so that the logo is in between the environment name and the search input.
- Release and click again and then drag. You'll see that you highlight the header element instead of dragging the service.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3780/
Test PASSed.

**Comment by hatched** [comment]
This is awesome @anthonydillon :+1: QA :ok:

File: jujugui/static/gui/src/app/assets/css/base.scss [hunk 1]
```diff
@@ -84,6 +84,33 @@ $left-panel-width: 290px;
 $navbar-height: 60px;
 $panel-space: 10px;
 
+// z-indexing
+$z-indexed-elements: search-results-list-block__tags,
```
Comment (hatched) [inline jujugui/static/gui/src/app/assets/css/base.scss]: What does this do?

**Comment by hatched** [comment]
I need to modify some of the same files so to avoid conflicts... :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1284

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**